workspaces/global-header/plugins/global-header/src/consts.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Color constants for the global header plugin
+ */
+export const COLORS = {
+  /**
+   * Gold color for starred items
+   */
+  STAR_GOLD: '#F3BA37',
```

Comment:
we should be defining colours in the theme package first and then consuming them in plugins, this way customers can customize these colours